cluster/gce/gci/configure-helper-common.sh

```diff
@@ -2313,8 +2320,8 @@ function apply-encryption-config() {
 #   DOCKER_REGISTRY
 function start-kube-controller-manager {
   echo "Start kubernetes controller-manager"
-  if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == "true" ]] || [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
-    create-kubeconfig "kube-controller-manager" ${KUBE_CONTROLLER_MANAGER_TOKEN} "${PROXY_RESERVED_IP}" "8888" "http"
+  if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" == "true" ]]; then
```

Comment:
here as well

Reply:
will address along with the decision 1) name of the variable to controller the insecure/secure port  2) two variables: one for enable insecure port, one for use insecure ports in kubemarktest

Reply:
fixed